manager.py

```diff
@@ -126,280 +70,269 @@ def readme_item(item):
+            coords.append(places[loc])
+        m["coordinates"] = coords
+
+
+def sub_coordinates(_):
+    mirrors = []
+    yaml = ruamel.yaml.YAML()
+    with open(SOURCE_YAML, encoding="utf-8") as source:
+        try:
+            mirrors = yaml.load(source)["archlinuxcn"]
+        except ruamel.yaml.YAMLError as error:
+            sys.exit(repr(error))
+    update_coordinates(mirrors)
+    with open(SOURCE_YAML, "w", encoding="utf-8") as output:
+        yaml.explicit_start = True
+        yaml.indent(mapping=2, sequence=4, offset=2)
+        yaml.dump({"archlinuxcn": mirrors}, output)
 
 
-def sub_geojson(args):
+def generate_geojson(mirrors):
     features = []
     geojson = {
         "type": "FeatureCollection",
         "features": features,
     }
-    with open(SOURCE_YAML, "r") as source:
-        try:
-            mirrors = ordered_load_yaml(source)
-            # mirrors.sort(key=lambda m: -mirror_score(m))
-        except yaml.YAMLError as e:
-            sys.exit(repr(e))
-
     for m in mirrors:
-        coords = m.get("geocoords")
-        if not coords:
+        locs = m.get("location")
+        coords = m.get("coordinates")
+        if locs is coords is None:
             continue
-        locs = m["geolocs"]
-        for loc, coord in zip(locs, coords):
-            lat, lon = coord.split(", ")
-            feature = {
-                "type": "Feature",
-                "properties": {
-                    "mirror": m["provider"],
-                    "url": m["url"],
-                    "name": loc,
-                },
-                "geometry": {
-                    "type": "Point",
-                    "coordinates": [float(lon), float(lat)],
-                },
-            }
-            features.append(feature)
-
-    with open(OUTPUT_GEOJSON, "w") as f:
-        json.dump(geojson, f, ensure_ascii=False)
+        for loc, coord in zip(locs, coords, strict=True):
+            lat, lon = map(float, coord.split(", "))
+            features.append(
+                {
+                    "type": "Feature",
+                    "properties": {
+                        "provider": m["provider"],
+                        "url": m["url"],
+                        "location": loc,
+                    },
+                    "geometry": {
+                        "type": "Point",
+                        "coordinates": [lon, lat],
+                    },
+                }
+            )
+    return geojson
+
+
+def sub_geojson(_):
+    with open(SOURCE_YAML, encoding="utf-8") as source:
+        try:
+            mirrors = ruamel.yaml.YAML(typ="safe").load(source)["archlinuxcn"]
+        except ruamel.yaml.YAMLError as error:
+            sys.exit(repr(error))
+    geojson = generate_geojson(mirrors)
+    with open(OUTPUT_GEOJSON, "w", encoding="utf-8") as output:
+        print(
+            re.sub(
+                r"(\[\n\s+)(.*)(,\n\s+)(.*)(\n\s+\])",
+                r"[\2, \4]",
```

Comment:
In order to format the generated geojson file according to Prettier’s formatting standards.